app/models/pame_evaluation.rb

```diff
@@ -1,7 +1,350 @@
+require 'csv'
+require 'wcmc_components'
+
 class PameEvaluation < ApplicationRecord
-  belongs_to :protected_area
+  include WcmcComponents::Loadable
+
+  belongs_to :protected_area, optional: true
   belongs_to :pame_source
   has_and_belongs_to_many :countries
 
-  validates :methodology, :year, :metadata_id, :url, presence: true
+  validates :methodology, :year, :metadata_id, presence: true
+
+  # config for loadable mixin
+  ignore_column :designation
+
+  import_by protected_area: :wdpa_id
+  import_by country: :iso_3
+  
+
+  TABLE_ATTRIBUTES = [
```

Comment:
Could we add `.freeze` to constants to make sure Rails throws an error if anything attemps to modify them?